CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## 2.9.2 - *2020-12-06*
 
 - resolved cookstyle error: libraries/filter_helper.rb:59:11 convention: `Style/CommentAnnotation`
+- Add a type to the EOL property of the filter_lines resource
```

Comment:
Hey @MarkGibbons 

This needs to go under the `## Unreleased` heading :) 

Thanks

Reply:
Should be fixed now.  thanks